kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md

```diff
@@ -58,137 +58,108 @@ The sarvaprāyaścitta was to be performed by the dying man or by his son or oth
 
-Rgveda X. 14,
+Ṛgveda X. 14,
 
 193
 
-our forefathers that went before us passed. May you (the departed) see the two kings Yama and god Varuna rejoicing as they will. 8. (O departed one!) be united in the highest heaven with the pitrs, with Yama and with your istāpurta; leaving here blamable acts (sins), come back to your homel Being endowed with bright lustre, be united with a (new) 448 body! 9 (Yo evil spirits ) Gọ away, depart, move aside from this place ( the cemetery), the pitrs made (or assigned) for him (the departed person ) this abode Yama gives to him (this) resting place characterized by waters, days and nights, 10. (O departed !) proceeding by a good path, hurry on past the two dappled dogs (of Yama), offspring of Saranā, each having four eyes. Thus approach the pitrs who will recognize 449 you and who enjoy delight in the company of Yama 11. O king Yama! Give him (the departed one) in charge of those two dogs of yours, who are guards, having four eyes each, who guard the path (to pitr-loka) and who watch men. May you bestow on him happiness and health! 1%. The two messengers of Yama, that have wide nostrils, that are very strong, that are satisfied with difficulty, move among the people. May those two (messengers) restore to us to-day auspicious life in order that we may see the Sun 1 450 13. Extract Soma juice for Yama (O priests !), offer oblation to Yama.451 Sacrifice, in which Agni is messenger (carrier) to the gods and which is well prepared reaches Yama, 14. (O priests) offer oblation sprinkled with ghee to Yama and then start. May he keep us (tied) to the 'worship of gods, 452 so that we may live a long life! 15. Offer to king Yama an oblation which is extremely sveet. Here is salutation to the sages who were born before us and who made the path for us,
 
 448 This appears to postulate the acquisition of a new etbereal body for the departed enabling him to enjoy the pleasures of pitrloka.
 
 449. 'Savıdatran'who will kaow or recogoize you. The Nir, V1.14 explains 'sayıdatrah Lalyānavidyāh'.
 
-450. The life implored for here is that of the persons related to the deceased who are left behind on the earth. 'Asutrpa' according to Sayana And Oldenberg means 'who steal away the lives of men'. This is a good mcading 10 view of what is stated so the last pada of the verse. In trags 1asing as done above the word is taken as 'a+su+trpa', while Sāyana takes it as 'asu +- trpa'.
+450. The life implored for here is that of the persons related to the deceased who are left behind on the earth. 'Asutrpa' according to Sayana And OldenbeṚg means 'who steal away the lives of men'. This is a good mcading 10 view of what is stated so the last pada of the verse. In trags 1asing as done above the word is taken as 'a+su+trpa', while Sāyana takes it as 'asu +- trpa'.
```

Comment:
Should be Oldenberg

Reply:
yes. i will correct that when i come to it.

Reply:
that was a wrong replacement. i will correct it.